README.md

```diff
@@ -11,11 +11,26 @@ repository. Whereas django-calaccess-raw-data supports only state-level data, we
 aim to add support for importing data from local jurisdictions who use Netfile
 to host their data.
 
+## Dependancies
+```
+python, pip, virtualenv
+Python 2.7.9 includes these:
+    Mac: brew install python
+postgresql:
+    Mac: brew install postgresql
+```
+
 ## Installation
 ```
 virtualenv env
 . env/bin/activate
 pip install -r requirements_dev.txt
+## create the database calaccess_raw
+mysql --user root 
+mysql> create database calaccess_raw;
+mysql> \q
```

Comment:
Your PR adds postgresql as a dependency on [line 19](https://github.com/opencalifornia/disclosure-backend/pull/3/files#diff-04c6e90faac2675aa89e2176d2eec7d8R19). Is it actually required to have both mysql and postgres installed? (I probably already have both on my machine, so maybe I didn't notice).

Even though calaccess_raw supports both mysql and postgresql, I thought we were going with postgres. I could be wrong, but that's what I read from the [May meeting notes](https://docs.google.com/document/d/1uLf1HtW4HCvLB3l8E64vpk6b_P9DTxiFNaxo1Sy5t2U/edit#heading=h.ugpb38ewhkx)